src/python_be.cc

```diff
@@ -1759,6 +1767,26 @@ ModelState::ModelState(TRITONBACKEND_Model* triton_model)
   }
 }
 
+std::string
+ModelState::DefaultArtifactName()
+{
+  if (platform_ == "tensorflow_savedmodel") {
+    return "model.savedmodel";
+  } else {
+    return "model.py";
+  }
+}
+
+std::string
+ModelState::PluginModel()
+{
+  if (platform_ == "tensorflow_savedmodel") {
```

Comment:
Is there a way that this could be managed by the platform_model.py and not the python_be?

Ideally the python_be wouldn't need to know any details of how the platform_model_loader works

Reply:
I think it will still be good to do a check whether a platform_model exists in the backend. If yes, then use the model.py from the platform_models or else fallback to previous pipeline.
This is to maintain backwards compatibility.